vendor-bin/test/composer.json

```
@@ -1,5 +1,5 @@
 {
     "require": {
-        "becklyn/php-cs": "^3.0"
+        "becklyn/php-cs": "^8"
```

```suggestion
        "becklyn/php-cs": ">=8"
```

We want to always stay up-to-date